tock/tock/urls.py

```diff
@@ -1,10 +1,22 @@
 from django.conf import settings
 from django.conf.urls import include, url
+from django.contrib.auth.decorators import user_passes_test
+from django.core.exceptions import PermissionDenied
 
 # Enable the Django admin.
 from django.contrib import admin
 admin.autodiscover()
 
+def check_if_staff(user):
+    if not user.is_authenticated():
+        return False
+    if user.is_staff:
```

Comment:
If you wanted, you could just say is user.is_authenticated() and user.is_staff. But I'm not sure you need user.is_authenticated() at all, since I don't think user.is_staff can be true unless they're authenticated and it can check.